components/footer/Footer.tsx

```diff
@@ -8,13 +8,13 @@ const Footer = () => {
     <div>
       <div className="bg-[#7a7c7b] w-full h-[50px] flex items-center justify-center">
         <div className="flex gap-4">
```

Comment:
i think it's here for fixing the footer icons on top of mobile nav bar

Reply:
its because i made the z-index 50 because it would work with anything else lower so thats probably why